zebrad/tests/common/lightwalletd.rs

```diff
@@ -266,12 +266,12 @@ impl LightwalletdTestType {
     }
 
     /// Returns the Zebra state path for this test, if set.
-    pub fn zebrad_state_path(&self) -> Option<PathBuf> {
+    pub fn zebrad_state_path(&self, test_name: String) -> Option<PathBuf> {
         match env::var_os(ZEBRA_CACHED_STATE_DIR_VAR) {
             Some(path) => Some(path.into()),
             None => {
                 tracing::info!(
-                    "skipped {self:?} lightwalletd test, \
+                    "skipped {test_name:?} lightwalletd test, \
```

Comment:
In the `lightwalletd_integration_test`, we run all 3 test types, so we need to log the type here too:

```suggestion
                    "skipped {test_name:?} {self:?} lightwalletd test, \
```